pyodide_lock/utils.py

```diff
@@ -103,7 +103,7 @@ def _generate_package_hash(full_path: Path) -> str:
 
 
 def _get_marker_environment(
-    platform: str, version: str, arch: str, python: str
+    platform: str, version: str, arch: str, python: str, abi_version: str | None = None
```

Comment:
The parameter doesn't seem to be used anywhere?

Reply:
But it gets passed and will crash if we don't accept it.

Reply:
I see... we should probably update the contest.py file then.